### ISSUE TITLE
Command "ClearCase: Find checkouts" stopped working

### ISSUE THREAD

**Issue by BillDenton** [OpenNingia/vscode-clearcase#36]
Ctrl-Shift-P followed by "ClearCase: Find checkouts" stopped working. Error message "command 'extension.ccFindCheckouts' not found" is displayed. On the previous version of the extension, it worked.

**Comment by OpenNingia** [comment]
I think the idea was to list the checkout files in the SCM tab on VSCode. However I also think that the old behavior that launched the Clearcase dialog should be preserved.

**Comment by BillDenton** [comment]
Oh. I didn't know about the Source Control Tab. I've looked at it and it showing my checked out files plus a huge list of view private files. Is there any way of filtering the view private files (like .gitignore)?
The Source Control tab looks better for general use, but I still it would be useful to be able to launch the "ClearCase Find checkouts" as well. It is showing checkin, compare with previous version, undo checkout and version tree, which are all the options I think I need.

**Comment by OpenNingia** [comment]
@BillDenton int the next (v2.0.3) version the Find Checkout command should work again. About the "ignore" filter on the SCM view please open another issue :)

**Comment by BillDenton** [comment]
I've also noticed that the view private list is showing items that I have added to ClearCase. When I updated the extension these files didn't exist. I then created them and added them to ClearCase (outside VS Code). I opened the SCM tab after they were added to ClearCase.
I'll add another issue for the filter. I've noticed there is a "View Private File Suffixes". However, is still showing a lot of other files (e.g. .obj, .exe, .bak, .vscode directory)
